.storybook/config.js

```diff
@@ -1,6 +1,6 @@
 import React, { Fragment } from 'react';
 import { configure, addDecorator, addParameters } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
-import { create } from '@storybook/theming';
+import { create } from '@storybook/theming'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
If eslint is complaining about importing dev dependencies - it does support config options that will allow these imports, or allow them in specific globs:

https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md

Reply:
@ryanmccombe great to know, can you open an issue and if you wish mini-PR to implement this across the repo

Reply:
https://github.com/bbc/simorgh/issues/1886